test/terra/utils/qasm_simulator/qasm_noncliffords.py

```diff
@@ -80,15 +80,15 @@ def test_ccx_gate_deterministic_default_basis_gates(self):
         self.is_completed(result)
         self.compare_counts(result, circuits, targets, delta=0)
 
-    def test_ccx_gate_nondeterministic_default_basis_gates(self):
-        """Test ccx-gate circuits compiling to backend default basis_gates."""
-        shots = 2000
-        circuits = ref_non_clifford.ccx_gate_circuits_nondeterministic(final_measure=True)
-        targets = ref_non_clifford.ccx_gate_counts_nondeterministic(shots)
-        qobj = compile(circuits, self.SIMULATOR, shots=shots)
-        result = self.SIMULATOR.run(qobj).result()
-        self.is_completed(result)
-        self.compare_counts(result, circuits, targets, delta=0.05 * shots)
+#     def test_ccx_gate_nondeterministic_default_basis_gates(self):
```

Comment:
Why are we removing this test?

Reply:
this test was occasionally failed. Please see the travis test in e5a9916, which does not change any in aer-impl.

Reply:
sorry. I modified a wrong file. I will reset this and push another change.